ckanext/unaids/react/components/FileInputComponent/src/FileUploader.js

```diff
@@ -1,37 +1,55 @@
 import React from 'react';
 import { useDropzone } from 'react-dropzone'
+import axios from 'axios';
 import { Client } from "giftless-client";
 
 export default function FileUploader({
-    maxResourceSize, lfsServer, orgId, datasetId, authToken,
+    maxResourceSize, lfsServer, orgId, datasetId,
     setUploadProgress, setUploadFileName, setHiddenInputs,
     setUploadError
 }) {
 
+    const getAuthToken = () =>
+        axios.post(
+            '/api/3/action/authz_authorize',
+            { scopes: `obj:ckan/${datasetId}/*:write` },
+            { withCredentials: true }
+        ).then(res => res.data.result.token)
```

Comment:
I think you're not handling `authz_authorize` error anywhere. Or is it going to be caught by:
``` 
            .catch(error => {
                setUploadError({
                    error: ckan.i18n._('Server Error'),
                    description: ckan.i18n._('An unknown server error has occurred.')
                });
                throw error;
            })
```
later down the line?

Reply:
Yep that's the case

Reply:
@Manoj-nathwani Why not do it in the same way as in batch upload then?
![image](https://user-images.githubusercontent.com/30174560/115016420-4a6f2800-9eb5-11eb-86a7-874985c50665.png)